### PR TITLE
update sdk due to billing plan swagger file change

### PR DESCRIPTION
**Pull request by Only2125**
reservation has a swagger change for billing plan. need to update sdk.

### CODE REVIEW

File: sdk/reservations/Microsoft.Azure.Management.Reservations/src/Microsoft.Azure.Management.Reservations.csproj [hunk 1]
```diff
@@ -7,6 +7,10 @@
 		<PackageId>Microsoft.Azure.Management.Reservations</PackageId>
 		<Description>Microsoft.Azure.Management.Reservations Library</Description>
 		<AssemblyName>Microsoft.Azure.Management.Reservations</AssemblyName>
+		<Version>1.13.0-preview</Version>
+    <PackageTags>Reservations;</PackageTags>
+		<PackageReleaseNotes>
+			Update billing plan sdk
 		<Version>1.12.0-preview</Version>
     <PackageTags>Reservations;</PackageTags>
 		<PackageReleaseNotes>
```
Comment (isra-fel) [inline sdk/reservations/Microsoft.Azure.Management.Reservations/src/Microsoft.Azure.Management.Reservations.csproj]: Wrong format -- this is why the build failed

File: src/SDKs/_metadata/reservations_resource-manager.txt [hunk 2]
```diff
@@ -0,0 +1,11 @@
+2019-10-28 03:47:35 UTC
```
Comment (isra-fel) [inline src/SDKs/_metadata/reservations_resource-manager.txt]: This file should not exist

**Review by isra-fel — CHANGES_REQUESTED** [review]
Hey @Only2125 please put a link to your swagger spec review PR in the comments

**Comment by Only2125** [comment]
> Hey @Only2125 please put a link to your swagger spec review PR in the comments

https://github.com/Azure/azure-rest-api-specs/pull/7541
Billing Plan swagger

File: src/SDKs/_metadata/reservations_resource-manager.txt (hunk 2 above)
Comment (Only2125) [reply src/SDKs/_metadata/reservations_resource-manager.txt]: Should I delete it manually? All it should never be generated?

Comment (isra-fel) [reply src/SDKs/_metadata/reservations_resource-manager.txt]: This file was generated by `generate.cmd` right? That is discouraged and we recommend `generate.ps1`.
So please remove this file manually, and use `generate.ps1` for all future code generation. Please remove `generate.cmd` as well. Thanks!

Comment (isra-fel) [reply src/SDKs/_metadata/reservations_resource-manager.txt]: Please address this issue as well, thanks!

Comment (Only2125) [reply src/SDKs/_metadata/reservations_resource-manager.txt]: deleted

**Review by isra-fel — APPROVED** [review]

**Pull request merged**